src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtUser.java

```diff
@@ -20,10 +22,15 @@
     String consent;
 
     /**
-     *  DigiTrust breaks the typical Prebid Server convention of namespacing "global" options inside "ext.prebid.*"
-     *  to match the recommendation from the broader digitrust community.
-     *  <p>
-     *  For more info, see: https://github.com/digi-trust/dt-cdn/wiki/OpenRTB-extension#openrtb-2x
+     * DigiTrust breaks the typical Prebid Server convention of namespacing "global" options inside "ext.prebid.*"
+     * to match the recommendation from the broader digitrust community.
+     * <p>
+     * For more info, see: https://github.com/digi-trust/dt-cdn/wiki/OpenRTB-extension#openrtb-2x
      */
     ExtUserDigiTrust digitrust;
+
+    /**
+     * A list of Universal ID modules that contain PubCommon ID and Unified ID
```

Comment:
Pls rephrase like `A list of Universal IDs that contain PubCommon ID and Unified ID`